packages/components/src/components/FileUploader/useFileUploader.ts

```diff
@@ -26,8 +26,12 @@ import {
 
 export type FileUploaderHookProps = {
   id: string | undefined;
-  /**Dersom komponenten styres utenfra. En liste over filer som skal være med ved første render. */
+  /**En liste over filer som skal være med ved første render. */
```

Comment:
Kan sikkert nevne at dette brukes i det tilfellet hvor `FileUploader` brukes uncontrolled. Kan kanskje samtidig endre beskrivelsen til å være mer beskrivende til formålet den skal brukes til 😄 
```suggestion
  /**Dersom komponenten skal styres internt. Utgangspunktet for filene som har blitt lastet opp. */
```